src/System.Xml.ReaderWriter/src/System/Xml/Core/XmlTextReaderImplAsync.cs

```diff
@@ -74,6 +76,43 @@ private Task FinishInitAsync()
             }
         }
 
+        private async Task FinishInitUriStringAsync()
+        {
+            // Substitutes removed XmlResolver.GetEntityAsync
+            Task<object> getEntityAsync = Task.Run(() =>
```

Comment:
Will change, thanks! I'm actually not sure if the tasks are run in separate threads. For Thread this is obvious, task is confusing to me.